Cargo.toml

```diff
@@ -9,6 +9,7 @@ edition = "2021"
 chrono = "0.4.38"
 clap = { version = "4.5.8", features = ["derive"] }
 inquire = { version = "0.7.5", features = ["date"] }
+clap_complete = "4.5.7"
```

Comment:
Looks like you've added this manually; use `cargo add clap_complete` instead.

Reply:
Okay sir